SimG4Core/Application/interface/Phase2SteppingAction.h

```diff
@@ -64,6 +64,7 @@ class Phase2SteppingAction : public G4UserSteppingAction {
   std::vector<const G4Region*> deadRegions;
   std::vector<G4LogicalVolume*> ekinVolumes;
   std::vector<int> ekinPDG;
+  G4String cmseName_, trackerName_, caloName_, btlName_;
```

Comment:
These variables are needed only for initialisation. In run time optimal would be to use G4String, but at initialisation better to use std::string, which would be easier to connect to other std methods.

Reply:
I shall do that when I remove dd4hep dependence